health/health.d/wmi.conf

```diff
@@ -137,3 +137,175 @@ component: Disk
     delay: down 15m multiplier 1.5 max 1h
      info: disk space utilization
        to: sysadmin
+
+### TCP
+# Source: https://learn.microsoft.com/en-us/troubleshoot/windows-server/networking/default-dynamic-port-range-tcpip-chang
+# Windows by default allows 16383 (Windows 2008 or newer) and 3976 (old versions) parallel ports.
+#
+# We are adopting conservative values.
+#
+ template: wmi_concurrent_tcp
+       on: wmi.tcp_conns_active
+    class: TCP
+     type: Windows
+component: IIS
+       os: *
+    hosts: *
+     calc: $ipv4 + $ipv6
+    units: connections
+    every: 10s
+     warn: $this > (($status >= $WARNING)  ? (2783) : (2385))
+     crit: $this > (($status == $CRITICAL) ? (3578) : (3180))
+     info: Number of ports used is reaching levels that won't accept more connections.
+       to: sysadmin
+
+ template: wmi_1m_ip_tcp_resets
+       on: wmi.tcp_conns_resets
+    class: Errors
+     type: Windows
+component: Network
+       os: *
+    hosts: *
+   lookup: average -1m at -10s unaligned absolute of ipv4,ipv6
+    units: tcp resets/s
+    every: 10s
+     info: average number of TCP RESETS over the last minute
+
+ template: wmi_10s_ip_tcp_resets
+       on: wmi.tcp_conns_active
+    class: Errors
+     type: System
+component: Network
+       os: *
+    hosts: *
+   lookup: average -10s unaligned absolute of ipv4,ipv6
+    units: tcp resets/s
+    every: 10s
+     warn: $netdata.uptime.uptime > (1 * 60) AND $this > ((($wmi_1m_ip_tcp_resets < 5)?(5):($wmi_1m_ip_tcp_resets)) * (($status >= $WARNING)  ? (1) : (10)))
```

Comment:
This doesn't work - using `wmi_1m_ip_tcp_resets` (ctx: `wmi.tcp_conns_resets`) in `wmi_10s_ip_tcp_resets` (ctx: `wmi.tcp_conns_active`) is wrong. I strongly suggest you check how health variables lookup works.

Reply:
We have only one chart per [host](https://github.com/netdata/go.d.plugin/blob/fe76173bd3dc40bdb100d624dfae7768a151752e/modules/wmi/charts.go#L514-L525), and familly variable is also unique as you can see [here](https://github.com/netdata/netdata/files/10332409/tcp_variables.txt).

I agree that for the first alerts there were a mistake, because we could have more instances and duplicated variables as we talked on Slack, but here it is a completely different scenario like we have for alert `tcp_connections`.

Reply:
> I strongly suggest you check how health variables lookup works.

Can you please do that? I tried to explain it several times, but as it turnes out - couldn't.

Reply:
After to refresh the mind I saw what was wrong :man_facepalming: , I am sorry for this.